content/blog/cats-case-study.md

```diff
@@ -0,0 +1,71 @@
+date = 2023-02-03T00:00:00Z
+description = ""
+draft = false
+image = "/images/2023/finance.jpg"
+slug = "finance-modernization-case-study"
+tags = ["Case Study", "Modernization"]
+title = "A Centralized System for a Leading Consumer Lending Bank "
++++
+
+### Client:
+
+A leading customer credit provider in the United States, committed to assisting customers in achieving their personal objectives through great customer service and flexible lending alternatives.
+
+### Problem statement:
+
+The bank has over 30 million unique loan contracts spread across various departments. As the number of contracts grew over time, handling all these contracts manually led to these major drawbacks.
+
+1. No centralized system to access all the contracts and work on them. The agents had to do multiple logins to different branch systems to work on these contracts and search across various branches even find the correct contract.
+2. While working on a contract and doing multiple operations, manual errors inevitably started creeping in.
+3. This led to the agents having to rework on the same contract and as a result, the turnaround time for a contract became extremely high across all the departments.
+
+### Solution:
+
+When the client first approached us, the goal was to transition one of the departments from the manual process to a digitized automated system. Once the initial problems were discovered, our team showcased the client various mockups for the product. After conducting research on various department workflows, we realized the huge potential involved in automation and cross-functional usability of the product.
+
+After the roll out to single department, there was such a huge success that everyone wanted to use it. There was no looking back from then onwards and the product transitioned the manual process in more and more departments.
+
+### Outcome:
+
+<span style="display: flex;flex-direction: row;align-content: center;justify-content: flex-start;align-items: center;">
+<h2 >~50%</h2> &nbsp;&nbsp; decrease in turnaround time per contract (1 hour to 30mins)
+</span>
+
+<span style="display: flex;flex-direction: row;align-content: center;justify-content: flex-start;align-items: center;">
+<h2>~4000</h2> &nbsp;&nbsp; manual hours saved every single day
+</span>
+
+<span style="display: flex;flex-direction: row;align-content: center;justify-content: flex-start;align-items: center;">
+<h2>~2X</h2> &nbsp;&nbsp; increase in contracts handled daily
+</span>
+
+<span style="display: flex;flex-direction: row;align-content: center;justify-content: flex-start;align-items: center;">
+<h2>~160x</h2> &nbsp;&nbsp; daily users across 5 different departments
+</span>
+
+### Challenges:
+
+1. **The client had been using an outdated CRM which was hosted on on-prem servers.** This involved huge maintenance costs for servers and a separate IT department. As the number of agents and contracts kept on increasing, cloud seemed to be the correct choice going forward.
```

Comment:
Is outdated proper word? Is there anything non offensive word? Like legacy?

Reply:
Using `legacy`